creepsy/README.md

```diff
@@ -13,6 +13,16 @@ To run this:
       "model": "viam-labs:audioout:pygame"
     }
 ```
+* Install the prerequisites to your Raspberry Pi:
+
+``` bash
+sudo apt update && sudo apt upgrade -y
+sudo apt-get install python3
+sudo apt install python3-pip
```

Comment:
Let's include the `python3-venv` package as well just to resolve the issue we saw with the audioout setup
```suggestion
sudo apt install python3-pip python3-venv
```